std/prettier/main.ts

```diff
@@ -343,10 +356,134 @@ async function* getTargetFiles(
   }
 }
 
+/**
+ * auto detect prettier configuration file and return config if file exist.
+ */
+async function autoResolveConfig(): Promise<PrettierBuildInOptions> {
+  const configFileNamesMap = {
+    ".prettierrc.json": 1,
+    ".prettierrc.yaml": 1,
+    ".prettierrc.yml": 1,
+    ".prettierrc.js": 1,
+    ".prettierrc.ts": 1,
+    "prettier.config.js": 1,
+    "prettier.config.ts": 1,
+    ".prettierrc.toml": 1
```

Comment:
Are all of these actually used? I’ve never seen a prettier.toml file before.

Reply:
yes. see https://prettier.io/docs/en/configuration.html